pkg/network/tracer/connection/ebpfless_tracer.go

```diff
@@ -240,49 +215,116 @@ func (t *ebpfLessTracer) processConnection(
 	switch result {
 	case ebpfless.ProcessResultNone:
 	case ebpfless.ProcessResultStoreConn:
+		if isNewConn {
+			conn.Duration = time.Duration(time.Now().UnixNano())
+			direction, err := t.determineConnectionDirection(conn, pktType)
```

Comment:
If we return unknown here, will we always get an error? Should we keep trying to determine the connection direction afterwards and not return an error here?